.gitignore

```diff
@@ -173,4 +173,10 @@ outputs-no-trainer/
 logs/
 logs-no-trainer/
 OUTPUT_DIR/
-test*.*
\ No newline at end of file
+mlruns/
+None/
+test*.*
```

Comment:
I'd like to ask some questions:
* What is this `None/` pattern about? What are you trying to ignore by it?
* `test*.*` is too broad of a pattern. Is it better just spelling out everything you want to ignore instead of this?

Reply:
When running mlflow, sometime it create None/ directory instead of mlruns/ . I believe it happens when something wrong with the tracking uri (not set, errors,..)